bite/host_vars/bite/lxc.yml

```diff
@@ -1,8 +1,8 @@
 lxc_host_containers:
   marvin:
-    state: started
+    state: stopped
   mediawiki:
-    state: started
+    state: stopped
```

Comment:
mulig det bare er jeg som ikke vet hvordan dette funker, men kjører disse to nå et annet sted?

Reply:
Niks.

Reply:
så de skal være `stopped` nå?

Reply:
men de skal være `stopped`?

Reply:
ja, det er riktig. De er ikke i bruk.